server/routes/api/accountManagement.js

```diff
@@ -169,4 +169,41 @@ module.exports = (app) => {
         });
       });
     }); //end of getDetails endpoint
-};
+
+  app.put('/api/account/:userID/basicInfo', verifyUser, function (req, res) {
+    // PUT http://localhost:8080/api/account/:userID/basicInfo
+    var user_id = req.params.userID;
+
+    //Verify that userID is present as a parameter
+    if (!user_id) {
+      return res.status(400).send({
+        success: false,
+        message: 'Error: userID parameter cannot be blank'
+      });
+    }
+
+    console.log("Request to update details of " + user_id);
+    var update = req.body
+
+    User.findOneAndUpdate(
+      {_id: user_id},
+      {basicInfo: Object.assign({},update)},
+      { new: true },
```

Comment:
What is new?
And can you change the updateAt field in timestamp?

Reply:
new returns the updated values back. Just realized that it isn't required.